<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="createAccountBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>创建账户</string>
   </property>
  </widget>
  <widget class="QComboBox" name="accountTypeComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QCheckBox" name="amountSortCheckBox">
      <property name="text">
       <string>按金额排序</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDateEdit" name="queryDateEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="queryBtn">
      <property name="text">
       <string>查询交易记录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>310</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QPushButton" name="queryStatsBtn">
      <property name="text">
       <string>统计收支</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDateEdit" name="statsDateEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>99</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>当前余额：0元</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="repayBtn">
      <property name="text">
       <string>还款</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="depositBtn">
      <property name="text">
       <string>存钱</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="withdrawBtn">
      <property name="text">
       <string>取钱</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="annualFeeEdit">
      <property name="placeholderText">
       <string>输入年费（仅信用账户）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="interestEdit">
      <property name="placeholderText">
       <string>输入利率（%）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
